service/kvmconfig/v6/key/key.go

```diff
@@ -43,8 +43,12 @@ const (
 	K8SKVMHealthDocker        = "quay.io/giantswarm/k8s-kvm-health:ddf211dfed52086ade32ab8c45e44eb0273319ef"
 	NodeControllerDockerImage = "quay.io/giantswarm/kvm-operator-node-controller:7146561e54142d4f986daee0206336ebee3ceb18"
 
-	// defaultWorkerMemory represents the extra memory to add due to qemu overhead.
-	defaultWorkerMemory = "1G"
+	// constants for calculation qemu memory overhead.
+	baseMasterMemoryOverhead     = "1G"
+	baseWorkerMemoryOverhead     = 512
```

Comment:
I would specify the unit in the type, e.g: baseWorkerMemoryOverheadMb

Reply:
done